<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="detalle_cto_certificados-20240607162539" tests="1" file=".py" time="60.345" timestamp="2024-06-07T16:26:40" failures="0" errors="1" skipped="0">
	<testcase classname="detalle_cto_certificados" name="test_detail_certificates" time="60.345" timestamp="2024-06-07T16:26:40" file="CtaCtoDtalCertificados.py" line="22">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=125.0.6422.142)
Stacktrace:
	GetHandleVerifier [0x00007FF7AABC1F52+60322]
	(No symbol) [0x00007FF7AAB3CEC9]
	(No symbol) [0x00007FF7AA9F7EBA]
	(No symbol) [0x00007FF7AA9CD5A5]
	(No symbol) [0x00007FF7AAA736B7]
	(No symbol) [0x00007FF7AAA8B331]
	(No symbol) [0x00007FF7AAA6BFC3]
	(No symbol) [0x00007FF7AAA39617]
	(No symbol) [0x00007FF7AAA3A211]
	GetHandleVerifier [0x00007FF7AAED94AD+3301629]
	GetHandleVerifier [0x00007FF7AAF236D3+3605283]
	GetHandleVerifier [0x00007FF7AAF19450+3563680]
	GetHandleVerifier [0x00007FF7AAC74326+790390]
	(No symbol) [0x00007FF7AAB4750F]
	(No symbol) [0x00007FF7AAB43404]
	(No symbol) [0x00007FF7AAB43592]
	(No symbol) [0x00007FF7AAB32F9F]
	BaseThreadInitThunk [0x00007FFBED37257D+29]
	RtlUserThreadStart [0x00007FFBEECEAA48+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\CtaCtoDtalCertificados.py", line 97, in test_detail_certificates
    validate_text_by_text(self.driver, text_expected)
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Elements.py", line 1000, in validate_text_by_text
    elemento = WebDriverWait(driver, 10).until(
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\expected_conditions.py", line 84, in _predicate
    return driver.find_element(*locator)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 742, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 348, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=125.0.6422.142)
Stacktrace:
	GetHandleVerifier [0x00007FF7AABC1F52+60322]
	(No symbol) [0x00007FF7AAB3CEC9]
	(No symbol) [0x00007FF7AA9F7EBA]
	(No symbol) [0x00007FF7AA9CD5A5]
	(No symbol) [0x00007FF7AAA736B7]
	(No symbol) [0x00007FF7AAA8B331]
	(No symbol) [0x00007FF7AAA6BFC3]
	(No symbol) [0x00007FF7AAA39617]
	(No symbol) [0x00007FF7AAA3A211]
	GetHandleVerifier [0x00007FF7AAED94AD+3301629]
	GetHandleVerifier [0x00007FF7AAF236D3+3605283]
	GetHandleVerifier [0x00007FF7AAF19450+3563680]
	GetHandleVerifier [0x00007FF7AAC74326+790390]
	(No symbol) [0x00007FF7AAB4750F]
	(No symbol) [0x00007FF7AAB43404]
	(No symbol) [0x00007FF7AAB43592]
	(No symbol) [0x00007FF7AAB32F9F]
	BaseThreadInitThunk [0x00007FFBED37257D+29]
	RtlUserThreadStart [0x00007FFBEECEAA48+40]

]]></error>
		<system-out><![CDATA[¡Elemento encontrado y clickeado 2 veces con éxito!
¡Elemento encontrado y clickeado 4 veces con éxito!
¡Elemento encontrado y clickeado 3 veces con éxito!
]]></system-out>
	</testcase>
</testsuite>
